core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -1373,10 +1373,29 @@ public UpdateCenter getUpdateCenter() {
         return updateCenter;
     }
 
+    /**
+     * If usage statistics has been disabled
+     *
+     * @since TODO
+     */
+    public Boolean isNoUsageStatistics() {
+        return noUsageStatistics;
+    }
+
+    /**
+     * If usage statistics are being collected
+     *
+     * @since TODO
+     */
     public boolean isUsageStatisticsCollected() {
         return noUsageStatistics==null || !noUsageStatistics;
     }
 
+    /**
+     * Sets the noUsageStatistics flag
+     *
+     * @since TODO
```

Comment:
It was introduced in 2011 or earlier, so it is not a big deal.  All reasonable  versions will be compatible https://github.com/jenkinsci/jenkins/commit/70c10658148c3eab3b4c4122705e1ed3a12e193d